features/backoffice_deregister_exemption.feature

```diff
@@ -0,0 +1,21 @@
+@backoffice
+Feature: Back office user deregisters a waste exemption activity from a registration
+  As a back office user
+  I want to deregister a waste exemption
+  So that the exemption is deregistered for a valid registration with mutlipe exemptions
```

Comment:
I think the point is that it there are more than one exemptions then it's only that exemption that is deregistered. Happy to change the wording though.